Fix publish pipeline

Error log:
```
npm ERR! package.json requires a "name" field

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-04-30T07_56_25_105Z-debug.log
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -11,9 +11,9 @@ jobs:
             - uses: actions/checkout@v1
             - uses: actions/setup-node@v1
               with:
-                  node-version: 12
+                  node-version: 13
                   registry-url: https://registry.npmjs.org/
-            - run: cd theme && npm install
-            - run: npm publish --access public
+            - name: "Install & publish"
+              run: cd theme && npm install && npm publish --access public
               env:
                   NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1]
+
+## Fixed
+
+-   Fix publish process
+
 ## [1.0.0]
 
 ### Added

--- a/theme/package.json
+++ b/theme/package.json
@@ -5,7 +5,7 @@
     "email": "admoni.bar1@gmail.com",
     "url": "https://github.com/baradm100"
   },
-  "version": "1.0.0",
+  "version": "1.0.1",
   "scripts": {
     "lint:ts": "tslint -c ../tslint.json 'theme/src/**/*.{ts,tsx}'",
     "lint:css": "stylelint --config ../.stylelintrc './src/**/*.{ts,tsx}'",

